src/main/docs/guide/configuration/availableOptions.adoc

```diff
@@ -1,5 +1,6 @@
 |===
 |`*micronaut.openapi.enabled*` | System property that enables or disables open api annotation processing. | Default: `true`
+|`*micronaut.openapi.swagger.file.generation.enabled*` | System property that enables or disables generation of a swagger (OpenAPI) specification file by code. Useful, when you want to use only views with already existed swagger file. | Default: `true`
```

Comment:
```suggestion
|`*micronaut.openapi.swagger.file.generation.enabled*` | System property that enables or disables generation of a swagger (OpenAPI) specification file. This can be used whenever you already have a specification file and that you only need the Swagger UI. | Default: `true`
```

Reply:
fixed